Source/CarthageKit/Git.swift

```diff
@@ -57,6 +57,30 @@ public struct GitURL: Equatable {
 		}
 	}
 
+	/// Amount of time before a git repository is fetched again. Defaults to 1 minute
+	public static var fetchExpirationTime: NSTimeInterval = 60.0
```

Comment:
This should probably have _interval_ in the name instead of _time_.

`fetchCacheInterval`?